calcom/cal.com
# Fix minimum booking notice set to 0

## What does this PR do?



This PR, solves the "Booking out of bounds" error when minimum notice was set to 0

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- In an event type, set the minimum booking notice to 0 under the "Limits" tab
- Go to the availability page. It should load

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
